docs/docs/protocol-specs/public-vm/_nested-context.md

```diff
@@ -6,24 +6,22 @@ The following shorthand syntax is used to refer to nested context derivation in
 // instr.args are { gasOffset, addrOffset, argsOffset, retOffset, retSize }
 
 isStaticCall = instr.opcode == STATICCALL
-isDelegateCall = instr.opcode == DELEGATECALL
 
-nestedContext = deriveContext(context, instr.args, isStaticCall, isDelegateCall)
+nestedContext = deriveContext(context, instr.args, isStaticCall)
 ```
 
 Nested context derivation is defined as follows:
 ```jsx
 nestedExecutionEnvironment = ExecutionEnvironment {
     address: M[addrOffset],
-    storageAddress: isDelegateCall ? context.storageAddress : M[addrOffset],
```

Comment:
Indeed, a related task is to remove storageAddress and move everything to address.

I'll send a message to aztec-3 about this.